addon/scripts/background.js

```diff
@@ -480,16 +556,26 @@ function badTransition(href, type, transitionType) {
         httpsRedirect[href] = false;
         return false;
     }
-    let maybeGood = (POTENTIALLY_GOOD_TRANSITIONS.indexOf(transitionType) >= 0);
+    let maybeGood = (VALID_TRANSITIONS.indexOf(transitionType) > -1);
     if (!type && !maybeGood) {
         return true;
     }
-    return type == SERVER_REDIRECT;
+    return BAD_TRANSITION.indexOf(type) > -1;
+}
+
+// Mark the url so that a sign doesn't occur again.
+function markSignUrl(url) {
+    return url + "&captcha-bypass=true";
```

Comment:
Is it possible to hand-made construct an URL with `&captcha-bypass=false` ? are there any side effects doing that?

Reply:
Yeah there are no side-effects to doing this, it would just cause the signing operation to happen again. The same as if the URL was intentionally stripped of the query string.